rules/sort-enums.ts

```diff
@@ -175,11 +174,18 @@ export default createEslintRule<Options, MESSAGE_ID>({
       )
 
       let sortingNodes = formattedMembers.flat()
-      let isNumericEnum = sortingNodes.every(
-        sortingNode =>
-          sortingNode.numericValue !== null &&
-          !Number.isNaN(sortingNode.numericValue),
-      )
+
+      let isNumericEnum = true
```

Comment:
[perf] Complexity is not changed here: `every` will early exit as soon as it encounters a condition that is `false`:

```ts
let func = (x) => {
    console.log("calling")
    return x === 0
}

[0, 0, 1, 0, 0, 0].every(func)
```
Result:
```ts
calling
calling
calling

Process finished with exit code 0
```

I personally think that the `every` statement is more concise and readable, but that's subjective.